ceph.spec.in

```diff
@@ -432,6 +432,7 @@ Group:          System/Filesystems
 Requires:       ceph-base = %{_epoch_prefix}%{version}-%{release}
 Requires:       python%{_python_buildid}-pecan
 Requires:       python%{_python_buildid}-six
+Requires:       python%{_python_buildid}-psutil
```

Comment:
This adds python-psutil to the runtime requires, i.e. it'll be installed when the package is installed, but that doesn't necessarily help on developer systems, where you run `./install-deps`, which pulls in only BuildRequires. So, this needs to *also* be added as BuildRequires to the spec file, inside one of the `%if 0%{with make_check}` guards.

It also also needs to be added to `debian/control`, probably in the main Build-Depends section, and in the Depends section for ceph-mgr (but someone with more debian packaging experience should probably double-check this :-/)

Reply:
Ok, added those packages to the BuildRequires and debian control file. I'm still not certain that this is the right thing to do though.

Reply:
It's arguably a bit weird, but we've done it before with other python dependencies required by the dashboard, so at least it's consistently a bit weird.